src/current/v23.1/backup-validation.md

```diff
@@ -15,6 +15,16 @@ You can validate a backup of a [cluster]({% link {{ page.version.version }}/back
 
 The options that give the most validation coverage will increase the runtime of the check. That is, `verify_backup_table_data` will take a longer time to validate a backup compared to `check_files` or `schema_only` alone. Despite that, each of these validation options provide a quicker way to validate a backup over running a "regular" restore.
 
+## Recommendations
+
+Cockroach Labs recommends implementing the following validation plan to test your backups:
+
+1. Very frequent [`schema_only`](#validate-a-backup-is-restorable) restores: Checks your credentials, schema issues, and version compatability.  
+1. Frequent [`verify_backup_table_data`](#validate-backup-table-data-is-restorable) restores: Tests that all data files are present and uncorrupted. 
+1. Somewhat frequent ["regular"]({% link {{ page.version.version }}/restore.md %}) restores: Full restores that completely validates a backup.  
```

Comment:
needs a rewording here for grammar

Reply:
tightened the wording around here to show that only a full restore can completely validate a backup